fr.etnadji.dispremi.json

```diff
@@ -0,0 +1,64 @@
+{
+    "app-id" : "fr.etnadji.dispremi",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "dispremi",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--device=dri"
+    ],
+    "add-extensions" : {
```

Comment:
It would be better to bundle ffmpeg here, something along https://github.com/flathub/com.github.robertsanseries.ciano/blob/master/com.github.robertsanseries.ciano.json#L77 should do.